pkg/container/container.go

```diff
@@ -118,8 +120,18 @@ func (c *Container) containerError(theErr error, msg string) error {
 	return errors.Wrapf(theErr, msg)
 }
 
-func (c *Container) Execute(args string, stdin io.Reader) error {
-	if err := c.SetConfig("lxc.execute.cmd", args); err != nil {
+func (c *Container) Execute(args []string, stdin io.Reader) error {
+	jblob, err := json.Marshal(args)
+	if err != nil {
+		return err
+	}
+
+	base64.StdEncoding.EncodeToString([]byte(jblob))
```

Comment:
I'm confused.  This line does nothing, right?

Reply:
correct. probably left over.